utils/push_repocard_examples.py

```diff
@@ -51,14 +55,22 @@ def push_model_card_example(overwrite: bool) -> None:
     Card is pushed to https://huggingface.co/templates/model-card-example.
     """
 
-    card = ModelCard.from_template(ModelCardData())
+    template = jinja2.Template(ModelCard.default_template_path.read_text())
+    content = template.render(card_data="{}")
```

Comment:
maybe pass description var and include link to raw template along with perhaps some note that that's where the rendered template came from? https://raw.githubusercontent.com/huggingface/huggingface_hub/main/src/huggingface_hub/templates/modelcard_template.md

Reply:
WDYT @Wauplin ?

Reply:
I like the idea !

Reply:
do you want to push directly to this branch @nateraw?

Reply:
sure! I see you added a couple commits 11 mins ago - are you done? If so I can push to this branch directly

Reply:
yep i'm done!